### PR TITLE
Makes cargo less tedious

### PR DESCRIPTION
**Pull request by freerealestate**
[Boxstation](http://i.imgur.com/MvQETy6.png)
- Adds disposal chutes to the cargo bay which go to conveyors in the cargo foyer, delivery office and warehouse to speed up crate delivery and storage
- Adds disposal chutes to the cargo office and warehouse to speed up crate return and loading crates from the warehouse onto the shuttle
- Extends cargo bay conveyors into shuttle to limit crate back-up and speed up the loading and unloading process
- Adds more room for crates on the cargo loading conveyor (mainly for the initial load of crates scattered around cargo and the mining dock)

supply.dm
- Speeds up order requests through the supply computers by removing the unnecessary confirmation screen afterwards
- Removes the window for cargo staff to have to input a request reason
- Tidies up the supply computer read-outs a bit and makes them more consistent with each other
- Adds the cost of orders to the requested orders and confirmed orders screens, as well as the job title of the person who requested the order or confirmed it
- Shows the number of remaining supply points on the cargo staff's View Orders and View Requests menus so they can easily prioritise which requests to approve or remove and assign blame to whoever wasted all the points

Unfortunately I had to keep the confirmation screens after approving or rejecting a request and clearing the request list as nobody could tell me how to remove them while having the screen update automatically afterwards. It works when sending the shuttle, but not there for some reason.

shuttle.dm
- Adds a space between ETA minute readout and mins unit in shuttle.dm, and changed "In Transit" to "In transit" to make it read better in context

floors.dmi
- Adds a new plating sprite to compliment existing ones: yellow and black stripes on opposite sides of the tile

### CODE REVIEW

File: code/controllers/subsystem/shuttles/supply.dm [hunk 1]
```diff
@@ -287,7 +287,7 @@
 				temp += "<A href='?src=\ref[src];doorder=[supply_type]'>[N.name]</A> Cost: [N.cost]<BR>"		//the obj because it would get caught by the garbage
 	else if (href_list["doorder"])
 		if(world.time < reqtime)
-			say("[world.time - reqtime] seconds remaining until another requisition form may be printed.")
+			say("[round((reqtime - world.time)/10,0.1)] second\s remaining until another requisition form may be printed.")
```
Comment (duncathan) [inline code/controllers/subsystem/shuttles/supply.dm]: Is this supposed to be reqtime-world.time?

**Comment by Shadowlight213** [comment]
I kinda liked the reason thing. If a crate needed to go to a certain department, I'd put the department's name on the reason to remind myself where to send it.

**Comment by CPTANT** [comment]
Good changes, cargo needs love.

Personally I would also reduce the cargo shuttle transit time to 1 minute, as in my opinion it serves no purpose other then being a PITA.

I would also love it when cargo would be able to purchase raw materials such as uranium and diamonds at a very high price, so cargo can partly fill the gap when there is a high need for materials but no mining.

**Comment by Supermichael777** [comment]
i was going to say getting tackled to the ground every third round by sec broke up the monotony a lot but these are good changes, even for those with strong crate-fu.

File: code/controllers/subsystem/shuttles/supply.dm (hunk 1 above)
Comment (freerealestate) [inline code/controllers/subsystem/shuttles/supply.dm]: Nope, that gives you negative numbers

**Comment by Supermichael777** [comment]
looks like it doesn't like that conveyor belt in the shuttle.

File: code/controllers/subsystem/shuttles/supply.dm (hunk 1 above)
Comment (JJRcop) [inline code/controllers/subsystem/shuttles/supply.dm]: Either @duncathan said it backwards or @freerealestate didn't notice...

Comment (freerealestate) [inline code/controllers/subsystem/shuttles/supply.dm]: Well I assumed he was suggesting it was supposed to be the other way around, which would give you a negative number

**Comment by Dorsisdwarf** [comment]
Can you reduce the time-to-centcomm travel time?

Thats my main cause of cargo monotony.

**Comment by freerealestate** [comment]
yup
i'll do it later

File: code/controllers/subsystem/shuttles/supply.dm (hunk 1 above)
Comment (Anonus) [inline code/controllers/subsystem/shuttles/supply.dm]: Well, if it's not supposed to be reqtime-world.time you should fix it.

**Comment by Supermichael777** [comment]
too many belts

**Comment by freerealestate** [comment]
The belt snake is not for navigating, it's for automatically loading crates onto the shuttle
It's snake shaped so the bottom of the shuttle is filled without leaving any gaps between the belts that an arriving crate could spawn in
All the belts are designed so they don't have to be crossed with crates, or if they do so a switch is nearby to turn them off

**Comment by MrStonedOne** [comment]
Please make a changelog,

You can add it to the PR's body rather then as a file now, see here: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157

**Comment by duncathan** [comment]
@freerealestate
Fix conflicts and also please make it clear on whether that change I mentioned was intended

**Comment by duncathan** [comment]
Conflicts + no creator response for two weeks = closed PR

@freerealestate ping me if you fix it

**Comment by freerealestate** [comment]
ok

will get back to this within a couple of weeks

**Pull request closed without merging**